cli/standalone/binary.rs

```diff
@@ -904,6 +906,27 @@ impl<'a> DenoCompileBinaryWriter<'a> {
   }
 }
 
+fn get_denort_path(deno_exe: PathBuf) -> Option<OsString> {
+  let mut denort = deno_exe;
+  denort.set_file_name("denort");
```

Comment:
> Can you test this on Windows? I guess the rt file is called denort there too (without a file extension).

Good point:

```suggestion
  denort.set_file_name(if cfg!(windows) { "denort.exe" } else { "denort" });
```

Reply:
I assumed that `denort` was a binary, non-executable file that was used to link against. I didn't realize it was actually an executable file, now I'm curious A) what it does, and B) how you build the custom compiled artifact. 

Anyways, I've applied this suggestion and fixed the formatting.